examples/ibm-is-ng/main.tf

```diff
@@ -410,6 +410,46 @@ data "ibm_is_dedicated_host" "dhost" {
   host_group = data.ibm_is_dedicated_host_group.dgroup.id
 }
 
+resource "ibm_is_snapshot" "r_snapshot" {
+  name = "my-snapshot"
+  source_volume = ibm_is_instance.instance1.volume_attachments[0].volume_id
+}
+
+data "ibm_is_snapshot" "ds_snapshot" {
+	name = "my-snapshot"
+}
+
+data "ibm_is_snapshots" "ds_snapshots" {
+}
+
+resource "ibm_is_instance" "instance4" {
+  name    = "instance4"
+  image   = var.image
+  profile = var.profile
+  instance_type = "volume"
```

Comment:
Could you explain the need for this attribute?

Reply:
I had to remove this attribute to run the config. This would need a doc update I guess.

Reply:
If this example is specific to snapshots then `image = var.image` is not required